R/run_diagnostics.R

```diff
@@ -13,6 +13,29 @@
 
 run_diagnostics <- function(mydir, model_settings ){
 
+  # Check for Report file
+  model_dir <- file.path(mydir, paste0(model_settings$base_name))
+  OS <- "Mac" # don't know the version$os info for Mac
+  if(length(grep("linux", version$os)) > 0) OS <- "Linux"
+  if(length(grep("mingw", version$os)) > 0) OS <- "Windows"
+
+  if (!file.exists(file.path(model_dir, "Report.sso"))) {
+    orig_dir <- getwd()
```

Comment:
Lines 23-36 can be replaced by 
`run(dir = model_dir, exe = model_settings$exe, extras = model_settings$extras)`